packages/xarc-react-router/src/browser/react-router-browser.tsx

```diff
@@ -1,9 +1,9 @@
 import { SubAppDef, SubAppFeatureFactory } from "@xarc/subapp";
 import { Router, BrowserRouter } from "react-router-dom";
-import { createBrowserHistory } from "history";
-
+//import RouterHistory from "./router-history"
 import { ReactRouterFeatureOptions, _id, _subId } from "../common";
-
+import { createBrowserHistory } from "history";
+const reactRouterHistory = createBrowserHistory();
```

Comment:
only create it when used

Reply:
I have to have reactRouterHistory on top because reactRouterFeature called multiple times for each sub-app and this is a one time call as shared history object